docs/content/patterns/alz/UpdateToNewReleases/Update_from_release_2023-11-14.md

```diff
@@ -25,20 +25,20 @@ Updating from release [2023-11-14](../../Whats-New#2023-11-14) will require runn
 
   {{% include "PowerShell-ExecutionPolicy.md" %}}
 
-  **Generate a list of the resource IDs which would be deleted by this script:**
+  **Show output of what would happen if deletes executed:**
 
   ```powershell
-  ./Start-AMBAOldArpCleanup.ps1 -pseudoRootManagementGroup $pseudoRootManagementGroup -ReportOnly
+  ./Start-AMBAOldArpCleanup.ps1 -pseudoManagementGroup $pseudoRootManagementGroup -WhatIf
```

Comment:
Can you validate? Looks like the param of the script is `[string]$pseudoRootManagementGroup`

Reply:
Done